modules/dxp/apps/search-experiences/search-experiences-service/src/test/java/com/liferay/search/experiences/internal/suggestions/spi/SXPBlueprintSuggestionsContributorTest.java

```diff
@@ -84,7 +87,9 @@ public void testGetSuggestionsContributorResults() throws Exception {
 
 		_setUpAssetRendererFactoryRegistryUtil(
 			"Asset Renderer Title", "Asset Renderer Summary");
-		_setUpSearcher(totalHits);
+
+		SetUpSearcherMockData mockData = _setUpSearcher(totalHits);
```

Comment:
Move below _setUpSuggestionsContributorConfiguration to get closer to usage in for loop